docs/contributing/triaging.md

```diff
@@ -0,0 +1,149 @@
+
+- **documentation**: Use this label for issues related to documentation.
+
+- **library**: Use this label for issues related to core libraries or preparations for future plugin impacts.
+
+
+### Requests and Feedback
+
+- **enhancement**: Use this label for requests for new features or improvements to existing functionalities.
+
+- **feedback**:  Use this label for issues where feedback is requested from the team or our community.
+
+- **help**: Use this label for issues where additional help or contributions are needed.
+
+
+### Special Labels
+
+- **automation**: Use this label for issues related to the CI/CD pipeline or automation
+
+- **duplicate**: Use this label for issues that are duplicates of existing ones.
+
+- **good first issue** : Use this label to highlight issues good for newcomers who want to contribute to Earthaccess.
+
+- **hackathon**: Use this label for issues to be discussed during hackdays.
+
+- **wontfix**: Use this label for issues that won’t be addressed or fixed.
+
+
+## Linking Labels in GitHub Markdown
+
+When referencing a label in a GitHub issue or discussion, it will be  useful to link to the label page to provide additional context and help other members to quickly understand the issue's category.
+
+### Syntax
+
+To link to a label in GitHub Markdown, use the following syntax:
+
+```
+[label name](https://github.com/username/repository/labels/label-name)
+```
+
+### Example
+
+For example, to link to the "hackathon" label in the nsidc/earthaccess repository, you would use the following syntax:
+
+```
+[hackathon](https://github.com/nsidc/earthaccess/labels/hackathon)
+```
+
+!!! tip
+    When linking to labels, it is a good practice to use the label name as the link text, as shown in the example above. This makes it clear to others what label is being referenced, and helps to avoid confusion.
+
+## Close Issue as Not Planned
+
+This section will cover the guidelines for when to use the "Close Issue as Not Planned" label, and how to handle issues that are not planned or feasible.
+
+### When to Use This Label?
+
+Use the "Close Issue as Not Planned" label when:
+
+- An issue is not aligned with the project's goals or priorities.
+- An issue is not feasible to be addressed due to technical or resource constraints.
+- An issue is a duplicate of an existing issue that has already been addressed.
+
+When closing an issue as not planned, keep the following best practices in mind:
+
+- Provide a clear explanation as to why the issue is not planned or feasible.
+- Offer alternative solutions or workarounds, if possible.
+- Link to relevant documentation or resources, if applicable.
+
+## Discussions vs Issues
+
+This section would cover the guidelines for when to use discussions versus issues, and how to migrate between them.
+
+###  What are Discussions?
+
+Discussions are used for:
+
+- Brainstorming and idea generation.
+- Feature requests and feedback.
```

Comment:
```suggestion
- Project feedback.
```

I wanted to differentiate this from "Requesting changes or improvements" in the "What are issues?" section :)